docs/kubeconfig-file.md

```diff
@@ -84,15 +84,20 @@ If the contents of the kubernetes_auth file conflict with entries in .kubeconfig
 
 ## Loading and merging rules
 The rules for loading and merging the .kubeconfig files are straightforward, but there are a lot of them.  The final config is built in this order:
-  1.  Merge together the kubeconfig itself.  This is done with the following hierarchy and merge rules:
+  1.  Get the kubeconfig  from disk.  This is done with the following hierarchy and merge rules:
+      
 
+      If the CommandLineLocation (the value of the `kubeconfig` command line option) is set, use this file only.  No merging.  
```

Comment:
Should specify that only one instance of this flag is currently allowed. We could allow multiple --kubeconfig options in the future if we decide we need merging later.